facia-tool/public/js/modules/content-api.js

```diff
@@ -186,13 +195,18 @@ function decorateBatch (articles) {
         }
 
         results.forEach(function(result) {
-            var icc = internalContentCode(result);
+            var pageCode = internalPageCode(result),
+                icc = internalContentCode(result);
 
-            if(icc) {
-                cache.put('contentApi', icc, result);
+            if (pageCode) {
+                cache.put('contentApi', pageCode, result);
+                if (icc) {
```

Comment:
yes, articles that are already added to a front will still use the old code. New articles instead will use the new code. Once there are no more articles using the old code, I'll remove that.